src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/baselines/MsftToSbSdkFiles.diff

```diff
@@ -262,6 +262,14 @@ index ------------
  ./sdk/x.y.z/Containers/tasks/netx.y/MSBuild.dll
  ./sdk/x.y.z/Containers/tasks/netx.y/Newtonsoft.Json.dll
  ./sdk/x.y.z/Containers/tasks/netx.y/NuGet.Common.dll
+@@ ------------ @@
+ ./sdk/x.y.z/DotnetTools/dotnet-format/Microsoft.CodeAnalysis.CSharp.Features.dll
+ ./sdk/x.y.z/DotnetTools/dotnet-format/Microsoft.CodeAnalysis.CSharp.Workspaces.dll
+ ./sdk/x.y.z/DotnetTools/dotnet-format/Microsoft.CodeAnalysis.dll
+-./sdk/x.y.z/DotnetTools/dotnet-format/Microsoft.CodeAnalysis.ExternalAccess.RazorCompiler.dll
```

Comment:
Since this is not intended to be the normal state, I would add this as an exclusion as well, pointing to https://github.com/dotnet/source-build/issues/4016#issuecomment-1908996093.